docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  bot:
+    image: krankkkk/dev-tj:latest
+    #image: togetherjava/tjbot:latest
+    volumes:
+      - "./application/config.json:/config.json"
+      - "./application/logs/:/logs"
+    links:
+      - logs
+    environment:
+      - TJ_LOG_TARGET=logs
+  logs:
+    image: krankkkk/dev-logviewer:latest
+    #image: togetherjava/tjlogs:latest
+    ports:
+      - "8080:8080"
```

Comment:
`8080` might already be in use by another application, it is too commonly used. Prefer another port please.

Reply:
Refractoring to `5050` then